### PR TITLE
Inkscape

### PR DESCRIPTION
**Pull request by jladan**
I'm starting the pull-request so it can be reviewed, but I still have some holes to fill, and maybe supplementary content.

The lesson.md file has some custom styles and `html` in it so that it renders well on the `https://uoftcoders.github.io/lessons/misc/inkscape-intro/lesson/` site.

A pre-rendered version of the latest can be seen on 
[my fork](https://jladan.github.io/studyGroup/lessons/misc/inkscape-intro/lesson/) of the site.

### CODE REVIEW

File: lessons/misc/inkscape-intro/lesson.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+- Escape will change tools and/or deselect. Keep pressing it until you're back at the selection tool with nothing selected.
+- The number row handles zooming 
+  - `=`, `-` zoom in and out
+  - `5` zooms to fit document
+  - `1` goes 1:1
+  - back-tick (`) goes to the previous zoom (like undo) 
+- ctrl-z for undo
+- ctrl-d for duplicate
+- alt-d for clone
```
Comment (QuLogic) [inline lessons/misc/inkscape-intro/lesson.md]: And what's the difference between these two?

File: lessons/misc/inkscape-intro/lesson.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+```
+  <group
+     id="extensions"
+     python-interpreter="/usr/bin/python2.7"
+     ...
+  >
+```
+
+The `plot function` extension also depended on the `lxml` module, which can be installed with `pip` (`pip2` you you have multiple versions of Python).
```
Comment (QuLogic) [inline lessons/misc/inkscape-intro/lesson.md]: s/you you/if you/

File: lessons/misc/inkscape-intro/lesson.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+</div>
+
+</div>
+
+
+
+
+## Python extensions
+
+There may be a problem related to python versions when running some extensions. If an extension has raises a syntax error, be sure to have Python 2.7 installed, and then specify the interpreter in `~/.config/inkscape/preferences.xml`
```
Comment (QuLogic) [inline lessons/misc/inkscape-intro/lesson.md]: s/has raises/raises/

File: lessons/misc/inkscape-intro/lesson.md (hunk 1 above)
Comment (jladan) [reply lessons/misc/inkscape-intro/lesson.md]: I'm not sure how to say it succinctly.  Best I have is "like duplicate, but the new object is linked to the original".

Changes to the original affect the clone, unless those properties were already changed when editing the clone.

**Comment by jladan** [comment]
Oh, and I should have said, looking at my schedule, it would be better to merge now, and I'll update the demos to show steps when I next have time.

**Comment by QuLogic** [comment]
BTW, you should probably add your waterloo email to github so it credits you properly.

**Comment by jladan** [comment]
D'oh! that's a typo. I guess it's too late to fix now.

**Pull request merged**